IPython/core/crashhandler.py

```diff
@@ -155,21 +163,23 @@ def __call__(self, etype, evalue, etb):
         color_scheme = 'NoColor'
 
         # Use this ONLY for developer debugging (keep commented out for release)
-        #color_scheme = 'Linux'   # dbg
-        try:
-            rptdir = self.app.ipython_dir
-        except:
+        # color_scheme = 'Linux'   # dbg
+        ipython_dir = getattr(self.app, "ipython_dir")
```

Comment:
getattr will raise AttributeError when property doesn't exist. Don't you want to provide None as the last argument?

@Carreau